components/Forum.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
We shouldn't put Forum into master in this state. We should wait until we have something with the forum functionality. I'd remove this from the pull request. I can show you how to do that with git.